go/private/go_toolchain.bzl

```diff
@@ -54,6 +54,8 @@ def _go_toolchain_impl(ctx):
 
         # Internal fields -- may be read by emit functions.
         _builder = ctx.executable.builder,
+        # TODO(yannic): Make this configurable.
+        _importpath_conflict_is_error = False,
```

Comment:
Let's make this a build flag instead of an option on the toolchain.

Add a `bool_flag` to `go/config/BUILD.bazel`. Make sure it's picked up by `//:go_config` and the `go_config` rule in `go/private/context.bzl`.

Reply:
Done